planning/testnet-kpis.md

```diff
@@ -1,29 +1,40 @@
-| Testnet uptime | Captures how robust and ready for mainnet Obscuro is | DataDog avg:system.uptime{*} | Average Testnet uptime over the last 4 weeks||
-| Number of end user accounts| Captures users using testnet in general and testing new dApps and shows the pipeline from wallet extension download to actively using testnet | TBD | Number of new end user account addresses in the last 4 weeks ||
-|DApps in development| This is difficult to measure as it’s not something we can look to the network to accurately measure. We can use our business development CRM data as an indicator. | Airtable "Partnerships" tab| Number of partners with status changed to "Soft Commitment" in the last 4 weeks||
-| New dApps deployed on Testnet| Indicates how much new activity and commitment of effort from dApp builders| ObscuroScan?| Number of new dApp addresses deployed in the last 4 weeks||
-| Wallet extension downloads| Good proxy for the number of active users. Straightforward to capture.| GitHub|Number of wallet extension downloads in the last 4 weeks||
+| Testnet uptime | Captures how robust and ready for mainnet Obscuro is | DataDog avg:system.uptime{*} | Average Testnet uptime over the last 4 weeks|99%|
+| Number of wallets connected to Obscuro Gateway| Good proxy for the number of active users. Straightforward to capture.| Datadog? |Number of daily connections|500|
+| Number of transactions| Typical guage of the amount of activity on testnet (even though it can be gamed).| Datadog? |Number of daily transactions|2000|
+
+### Selected KPIs for Developers
+| KPI NAME | RATIONALE | SOURCE | METRIC | TARGET |
+|--|--|--|--|--|
+| Testnet documentation page views (https://docs.obscu.ro/testnet/) | Gauges interest from developers in building on testnet. A good early indicator. Straightforward to capture | Google Analytics | Number of unique page views in the last 4 weeks |50|
+|DApps in development| This is difficult to measure as it’s not something we can look to the network to accurately measure. We can use our business development CRM data as an indicator. | Airtable "Partnerships" tab| Number of partners with status changed to "Soft Commitment" in the last 4 weeks|40|
+| New dApps deployed on Testnet| Indicates how much new activity and commitment of effort from dApp builders| ObscuroScan?| Number of new dApp addresses deployed in the last 4 weeks|20|
 
 ## Discounted KPIs
-These KPIs were considered and discounted.
+These KPIs were considered and discounted for now.
 
+- Wallet extension downloads
+    - Good proxy for the number of active users.
+    - Straightforward to capture.
+    - Discounted because Obscuro Gateway is the recommended wallet connection method.
```

Comment:
Agreed. It almost becomes a negative KPI. Hopefully the wallet extension gets downloaded 0 times which will show people trust our gateway.